packages/core/package.json

```diff
@@ -35,6 +35,7 @@
     "@truffle/debugger": "^11.0.22",
     "@truffle/decoder": "^5.3.5",
     "@truffle/deployer": "^3.3.2",
+    "@truffle/db-loader": "^0.2.10",
```

Comment:
Nice catching the libs we didn't have listed! This probably should have been in a separate PR though.

Reply:
this was causing issues in the CI check for this PR.